functions/openhab.bash

```diff
@@ -188,7 +188,7 @@ multitail_openhab_scheme() {
 ##    openhab_is_installed()
 ##
 openhab_is_installed() {
-  if dpkg -s 'openhab2' &> /dev/null; then return 0; else return 1; fi
+  if dpkg -s 'openhab' &> /dev/null; then return 0; else return 1; fi
```

Comment:
we should create a helper is_installed with the package(s) being an argument you pass
dpkg is often in use e.g. in exim, wireguard, amanda etc IIRC

Reply:
perhaps, but that is outside of the scope of this pr. You should create an issue and I can work on a PR for master after this is done.

Reply:
right